pkgs/development/libraries/gnu-efi/default.nix

```diff
@@ -15,12 +15,12 @@ stdenv.mkDerivation rec {
 
   makeFlags = [
     "PREFIX=\${out}"
-    "CC=gcc"
-    "AS=as"
-    "LD=ld"
-    "AR=ar"
-    "RANLIB=ranlib"
-    "OBJCOPY=objcopy"
+    "CC=${stdenv.cc.prefix}gcc"
```

Comment:
Shouldn't this be `cc instead of `gcc` to be more clang-friendly?

Reply:
Indeed it should.